packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -503,6 +594,107 @@ test.serial('writeGranulesFromMessage() saves the same values to DynamoDB, Postg
   t.deepEqual(omit(translatedPgRecord, omitList), omit(esRecord, omitList));
 });
 
+test.serial('writeGranulesFromMessage() given a partial granule updates only provided fields', async (t) => {
+  const {
+    collectionCumulusId,
+    cumulusMessage,
+    esGranulesClient,
+    granuleModel,
+    granulePgModel,
+    knex,
+    executionCumulusId,
+    providerCumulusId,
+    granuleId,
+  } = t.context;
+
+  await writeGranulesFromMessage({
+    cumulusMessage,
+    executionCumulusId,
+    providerCumulusId,
+    knex,
+    granuleModel,
+  });
+
+  t.true(await granuleModel.exists({ granuleId }));
```

Comment:
Quibble, this isn't validating the initial state, just existance.    If the premise of the test is to validate upsert happens as expected we should probably be explicit re: what the record is.

Reply:
Edit: I see, looking at the next test you're doing it there. 

The composition works here, although I have to wonder if there's value in seperate test stanzas/assertions, but it's a nit.

Reply:
Yeah, this is another "Did the write happen?" check. The intention is just, given the `writeGranulesFromMessage` call, do we now have a granule in all of our datastores? Then we update it and do a more thorough comparison. That's the intention anyway.